src/anyPass.js

```diff
@@ -1,5 +1,5 @@
-var _any = require('./internal/_any');
```

Comment:
IDK, I kind of think keeping internal versions of these make sense (for performance and future extensions)